components/Header/MobileHeader.vue

```diff
@@ -38,6 +39,14 @@
             />
           </div>
 
+          <!--
```

Comment:
Lets try to keep from adding just commented out code. If it's necessary to add for later, then please document with comments as to why